cves/CVE-2013-0848.yml

```diff
@@ -205,8 +213,9 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: true
+  answer: This vulnerablity allows the system to be able to crash due to array out of bounds index. This violates
+  the requirement that the system should not crash.
```

Comment:
I didn't think of this field this way as I took it more of video codec standards, but I think this is a valid answer. Not sure if there are any FFmpeg goals that explicitly state this but I think it's assumed for most software. Nice job!

Reply:
Also another issue with `|` versus one line.

Reply:
I would say its a stretch to call "the system should not crash" a specification